content/en/docs/concepts/workloads/pods/init-containers.md

```diff
@@ -289,72 +288,6 @@ The Pod which is already running correctly would be killed by `activeDeadlineSec
-allows you to specify a `restartPolicy` for init containers which is independent of
-the Pod and other init containers. Container [probes](/docs/concepts/workloads/pods/pod-lifecycle/#types-of-probe)
-can also be added to control their lifecycle.
-
-If an init container is created with its `restartPolicy` set to `Always`, it will
-start and remain running during the entire life of the Pod, which is useful for
-running supporting services separated from the main application containers.
-
-If a `readinessProbe` is specified for this init container, its result will be used
-to determine the `ready` state of the Pod.
-
-Since these containers are defined as init containers, they benefit from the same
-ordering and sequential guarantees as other init containers, allowing them to
-be mixed with other init containers into complex Pod initialization flows.
-
-Compared to regular init containers, sidecar-style init containers continue to
-run and the next init container can begin starting once the kubelet has set
-the `started` container status for the sidecar-style init container to true.
-That status either becomes true because there is a process running in the
-container and no startup probe defined, or
-as a result of its `startupProbe` succeeding.
-
-This feature can be used to implement the sidecar container pattern in a more
-robust way, as the kubelet always restarts a sidecar container if it fails.
-
-Here's an example of a Deployment with two containers, one of which is a sidecar:
-
-{{% code_sample language="yaml" file="application/deployment-sidecar.yaml" %}}
-
-This feature is also useful for running Jobs with sidecars, as the sidecar
-container will not prevent the Job from completing after the main container
-has finished.
-
-Here's an example of a Job with two containers, one of which is a sidecar:
-
-{{% code_sample language="yaml" file="application/job/job-sidecar.yaml" %}}
-
-#### Resource sharing within containers
-
-Given the ordering and execution for init containers, the following rules
-for resource usage apply:
-
-* The highest of any particular resource request or limit defined on all init
-  containers is the *effective init request/limit*. If any resource has no
-  resource limit specified this is considered as the highest limit.
-* The Pod's *effective request/limit* for a resource is the higher of:
-  * the sum of all app containers request/limit for a resource
-  * the effective init request/limit for a resource
-* Scheduling is done based on effective requests/limits, which means
-  init containers can reserve resources for initialization that are not used
-  during the life of the Pod.
-* The QoS (quality of service) tier of the Pod's *effective QoS tier* is the
-  QoS tier for init containers and app containers alike.
-
-Quota and limits are applied based on the effective Pod request and
-limit.
-
-Pod level control groups (cgroups) are based on the effective Pod request and
-limit, the same as the scheduler.
```

Comment:
I would leave these line here, along with a note that they also appear in the sidecar containers page (so people know to update both places).

Reply:
Is it really worth of repeating the same section in both the pages? 

I feel, instead of mentioning the same text in both the pages we can keep this in [init containers](/docs/content/en/docs/concepts/workloads/pods/init-containers.md) page and provide a hyperlink in [sidecar container](/docs/concepts/workloads/pods/sidecar-containers/) page to redirect people?

@sftim,
WDYT!

Reply:
Here, i keep these lines as it is under [init containers](/docs/concepts/workloads/pods/init-containers/#resource-sharing-within-containers) page and provided a hyperlink in [sidecar containers](/docs/concepts/workloads/pods/sidecar-containers/#resource-sharing-within-containers) page as mentioned above comment, Please let me know if it is not accurate. i will revert back and apply your suggestions.

Reply:
> Is it really worth of repeating the same section in both the pages?

I think it is worth the effort. Although it is more maintenance work, there are many more readers than docs contributors. The duplication is valuable to readers and so the payoff has a big multiplier.